src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.ts

```diff
@@ -41,6 +41,15 @@ export class StudentExamDetailComponent implements OnInit {
     isBonus = false;
     passed = false;
 
+    workingTimeFormValues = {
+        hours: 0,
+        minutes: 0,
+        seconds: 0,
+        percent: 0,
+    };
+
+    private normalWorkingTime = 0;
```

Comment:
I think it *might* be confusing that this field has the same name as the import in line 17. Coudl we find a slightly different name? Mabye ``currentNormalWorkingTime`` or ``regularWorkingTime`` (I think you used that term already somewhere below in another file)?

Reply:
I renamed it to `regularExamWorkingTime` now.